lua/telescope/_extensions/bibtex.lua

```diff
@@ -68,6 +82,10 @@ end
 
 local function bibtex_picker(opts)
   opts = opts or {}
+  if not files_initialized then
```

Comment:
FWIW because you already merged :rofl: You could initialize files as nil.
```lua
local files -- nil at that point
...
local function bibtex_picker
  opts = opts or {]
  if files == nil then
    initFiles()
  end
...
```

Doesn't really matter just saves one var :)